### PR TITLE
invite post to use application/JSON

### PR DESCRIPTION
**Pull request by aaronkvanmeerten**
changed to .ajax from .post to allow content type to be set

### CODE REVIEW

**Comment by aaronkvanmeerten** [comment]
This is a fix that allows invitations to be sent as application/json rather than x-form-encoded content type.  It has not been tested at all, so do not merge without testing thoroughly please.

File: react/features/invite/functions.js [hunk 1]
```diff
@@ -275,15 +275,20 @@ export function invitePeopleAndChatRooms( // eslint-disable-line max-params
...
-        $.post(
-                `${inviteServiceUrl}?token=${jwt}`,
-                JSON.stringify({
-                    'invited': inviteItems,
-                    'url': inviteUrl
-                }),
-                resolve,
-                'json')
-            .fail((jqxhr, textStatus, error) => reject(error));
+        $.ajax({
```
Comment (virtuacoplenny) [inline react/features/invite/functions.js]: If you're changing stuff, maybe you should just make it use fetch?

Comment (virtuacoplenny) [inline react/features/invite/functions.js]: If you're changing stuff and testing with this much change, maybe you should just make it use fetch?

Comment (aaronkvanmeerten) [reply react/features/invite/functions.js]: done!

**Comment by aaronkvanmeerten** [comment]
Tested this change and it appears to work as expected.

**Pull request merged**